string object_name
AnnotationPt2D[] control_points
